src/cart/crt_corpc.c

```diff
@@ -39,6 +39,7 @@ crt_corpc_info_init(struct crt_rpc_priv *rpc_priv,
 	struct crt_corpc_hdr	*co_hdr;
 	int			 rc;
 
+	DBG_ENTRY();
```

Comment:
if we are to add new symbol/define that is visible from outside of cart internals we should rename those into D_DBG_ENTRY()  and D_DBG_EXIT() or some such (D_ prefix for all externally visible macros, unless it is CRT macro in which case CRT_ prefix)

Reply:
Currently, DBG_EXIT and DBG_ENTER are defined in crt_internal.h, and was meant for cart.  I can change the name  or just remove them from the files.  You have final responsibility for this code so I'm leaving the decision up to you.  If we leave them in, then the default is to have each of these to be NULL.

Reply:
Change names to CRT_ENTER and CRT_EXIT.
They are defined in crt_internal.h.  By default, they are defined to nothing so no output is generated at run time.
No hit in timing.